torch/csrc/api/include/torch/nn/modules/dropout.h

```diff
@@ -23,26 +25,50 @@ class DropoutImplBase : public torch::nn::Cloneable<Derived> {
   explicit DropoutImplBase(DropoutOptions options_);
 
   void reset() override;
+
+  /// During training, applies a noise mask to the input tensor.
+  /// During evaluation, applies an identity function.
   Tensor forward(Tensor input);
```

Comment:
Not about this PR. I just want to mention that now dropout has been moved into ATen (and has efficient paths like fused CUDA kernels), maybe you could update the implementations of these Modules to call that. :)

Reply:
true should do